management-api-server/src/main/java/com/datastax/mgmtapi/resources/v2/TableOpsResources.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Please see the included license file for details.
+ */
+package com.datastax.mgmtapi.resources.v2;
+
+import com.datastax.mgmtapi.ManagementApplication;
+import com.datastax.mgmtapi.resources.common.BaseResources;
+import com.datastax.mgmtapi.resources.models.Table;
+import com.datastax.oss.driver.api.core.cql.ResultSet;
+import com.datastax.oss.driver.api.core.cql.Row;
+import com.google.common.collect.Lists;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.ExampleObject;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import java.util.List;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.http.HttpStatus;
+
+@Path("/api/v2/ops/tables")
+public class TableOpsResources extends BaseResources {
+
```

Comment:
Issue:
I think we can move the `list()` method you have in here to the v1 `TableOpsResources` class here:
https://github.com/k8ssandra/management-api-for-apache-cassandra/blob/feature/get-tables/management-api-server/src/main/java/com/datastax/mgmtapi/resources/v1/TableOpsResources.java
That v1 implementation doesn't already have a `list` operation. Moving this there would keep the v2 api space clean for now. Also, if you choose to not change the `operationId` below, then the generated client will create the method as `listTables`()`, which would be more clear that it's a v1 endpoint, not a v2 endpoint. However, I would recommend explicitly providing a unique operationId to make it clear this is a v1 endpoint.